spaceship/__init__.py

```diff
@@ -1,5 +1,23 @@
 from flask import Flask
+from flask_assets import Environment, Bundle
```

Comment:
to avoid polluting `__init__`, we should move all of this stuff into it's own file. for example, see `db.py`. we should definitely move asset initialization after we config the app via `app.config.from_object`.